skypy/galaxies/vdf_sampler.py

```diff
@@ -0,0 +1,92 @@
+"""Sample from velocity distribution function
+"""
+
+import numpy as np
+import scipy.special as special
+from scipy.interpolate import interp1d
+
+
+def sample_vdf(x_min, x_max, resolution=100, size=1):
+    """Sample from velocity dispersion function of elliptical galaxies in the local universe [1]_.
+
+    Parameters
```

Comment:
Update according to the other comments.

Reply:
> From the context I assume that you implement a sampler based on equation (4) of the Choi et al. paper. Please clarify this in the docstring.
> 
> I see that the velocity dispersion is simply a Schechter function, which is already implemented in SkyPy. I suggest that you follow the same syntax. I.e. add a module `velocity_dispersion.py` to the galaxy package and in this module add a function `schechter_vdf()`. You can find examples in [skypy.galaxies.luminosity](https://github.com/skypyproject/skypy/blob/main/skypy/galaxies/luminosity.py), [skypy.galaxies.redshift](https://github.com/skypyproject/skypy/blob/main/skypy/galaxies/redshift.py) and [skypy.galaxies.stellar_mass](https://github.com/skypyproject/skypy/blob/main/skypy/galaxies/stellar_mass.py).
> 
> We have also already implemented a sampler `schechter` to sample from the Schechter function in [skypy.utils.random](https://github.com/skypyproject/skypy/blob/df2536974c122dd72498f8d146913c14c832833b/skypy/utils/random.py#L21). You can use it in the same way as done in the examples mentioned above. In this case you don't need the class `PDFSampling` and the function `approx_cdf_1d()`.
> 
> I added some further comments about naming conventions at the specific lines in the codes.

Hi Philipp - thanks for the feedback. I have committed the changes. Please let me know what you think.